### PR TITLE
change deprecated package

### PR DESCRIPTION
**Pull request by BykhovDenis**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svele-check is applied (rush svelte check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

**Review by haiodo — DISMISSED** [review]

**Comment by aplatoff** [comment]
Denis, this cool

**Comment by aplatoff** [comment]
Please change something which I do not like

File: packages/ui/src/components/calendar/DateInputBox.svelte [hunk 1]
```diff
@@ -44,34 +44,17 @@
...
-              .valueOf()
-            : moment(date)
-              .month(val - 1)
-              .valueOf()
-        )
-        break
-      case 'year':
-        date = new Date(timeZone ? moment(date).tz(timeZone).year(val).valueOf() : moment(date).year(val).valueOf())
-        break
-      case 'hour':
```
Comment (aplatoff) [inline packages/ui/src/components/calendar/DateInputBox.svelte:66]: Do not like this

File: packages/ui/src/components/calendar/DateInputBox.svelte [hunk 2]
```diff
@@ -86,24 +69,14 @@
...
-      case 'day':
-        return timeZone ? moment(date).tz(timeZone).date() : moment(date).date()
-      case 'month':
-        return timeZone ? moment(date).tz(timeZone).month() + 1 : moment(date).month() + 1
-      case 'year':
-        return timeZone ? moment(date).tz(timeZone).year() : moment(date).year()
-      case 'hour':
-        return timeZone ? moment(date).tz(timeZone).hours() : moment(date).hours()
-      case 'min':
-        return timeZone ? moment(date).tz(timeZone).minutes() : moment(date).minutes()
```
Comment (aplatoff) [inline packages/ui/src/components/calendar/DateInputBox.svelte:105]: This is bad too

File: packages/ui/src/components/calendar/Month.svelte [hunk 3]
```diff
@@ -50,11 +50,11 @@
   let days: ICell[] = []
   const getDateStyle = (date: Date): TCellStyle => {
     if (currentDate != null) {
-      const zonedTime = moment(currentDate).tz(timeZone)
+      const zonedTime = DateTime.fromJSDate(currentDate).setZone(timeZone)
       if (
```
Comment (aplatoff) [inline packages/ui/src/components/calendar/Month.svelte:54]: Improve this

File: packages/ui/src/components/calendar/Month.svelte [hunk 4]
```diff
@@ -50,11 +50,11 @@
...
-        zonedTime.date() === date.getDate() &&
-        zonedTime.year() === date.getFullYear() &&
-        zonedTime.month() === date.getMonth()
+        zonedTime.day === date.getDate() &&
```
Comment (aplatoff) [inline packages/ui/src/components/calendar/Month.svelte:55]:
```suggestion
        zonedTime.day === date.getDate() &&
```

**Review by aplatoff — CHANGES_REQUESTED** [review]
Please fix

**Review by haiodo — APPROVED** [review]

File: packages/ui/src/components/calendar/DateInputBox.svelte (hunk 2 above)
Comment (aplatoff) [reply packages/ui/src/components/calendar/DateInputBox.svelte:105]: Translate to French

File: packages/ui/src/components/calendar/DateInputBox.svelte [hunk 5]
```diff
@@ -161,7 +134,7 @@
           selected = 'month'
         } else if (selected === 'month' && (shouldNext || edits[1].value > 1)) selected = 'year'
         else if (selected === 'year' && withTime && (shouldNext || edits[2].value > 999)) selected = 'hour'
-        else if (selected === 'hour' && (shouldNext || edits[3].value > 2)) selected = 'min'
+        else if (selected === 'hour' && (shouldNext || edits[3].value > 2)) selected = 'minute'
```
Comment (aplatoff) [inline packages/ui/src/components/calendar/DateInputBox.svelte:137]: Nice

**Comment by aplatoff** [comment]
Great Job!

File: packages/ui/src/components/calendar/Month.svelte (hunk 3 above)
Comment (haiodo) [reply packages/ui/src/components/calendar/Month.svelte:54]: Yes please improve $1

File: packages/ui/src/components/calendar/Month.svelte (hunk 4 above)
Comment (haiodo) [reply packages/ui/src/components/calendar/Month.svelte:55]: Please improve $2

**Review by haiodo — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/calendar/DateInputBox.svelte (hunk 5 above)
Comment (haiodo) [reply packages/ui/src/components/calendar/DateInputBox.svelte:137]: +21213

**Comment by haiodo** [comment]
@BykhovDenis did you plan to complete it?

**Pull request closed without merging**